database/migrations/2023_03_20_115209_create_messages_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('messages', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('doctor_id');
+            $table->foreignId('patient_id');
+            $table->string('message');
+            $table->file('file');
+            $table->int('type_id');
```

Comment:
maybe you wanted to put type_id as foreignId, maybe not. I do not know if there is difference